lib/Semantics/mod-file.cpp

```diff
@@ -719,8 +719,8 @@ static std::string CheckSum(const std::string_view &contents) {
   return result;
 }
 
-static bool VerifyHeader(const char *content, std::size_t len) {
-  std::string_view sv{content, len};
+static bool VerifyHeader(llvm::ArrayRef<char> content) {
```

Comment:
Why not just pass the sourceFile?

Reply:
Because we need to ignore the BOM and the carraige returns, so we need the slicing behaviour of `content()`